governance/votes/Executive vote - September 5, 2024.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Template - [Executive Vote] Spark Proxy Spell and Launch Project Funding - September 5, 2024
+summary: Trigger Spark Proxy Spell and transfer DAI and MKR to Launch Project. 
+date: 2024-09-05T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Spark Proxy Spell and Launch Project Funding - September 5, 2024
+
+The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

Comment:
Sidestream are crafting so should come first, add Phoenix Labs as they are overall responsible for Spark spell crafting. On further thought, should this be Wonderland as the Spark spell crafters?

```suggestion
The Governance Facilitators, Sidestream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

Reply:
@VoteWizard what do you think? I believe this is the first time Wonderland has fully crafted the proxy spell. Should we credit Phoenix Labs and Wonderland in this paragraph?

Reply:
Our view is that it should probably read "Sidestream, Dewiz, and Wonderland" as this would be consistent with previous Spark spells crafted by Phoenix labs and reviewed by other entities, although we are open to opposing views.

Reply:
As discussed in DMs, I agree with the suggestion, we should include them. Great suggestion!